### ISSUE TITLE
VK_DEVICE_LOST error when copying filtered environment

### ISSUE THREAD

**Issue by abwood** [KhronosGroup/glTF-IBL-Sampler#8]
I am trying to generate a prefiltered environment map for a [sample HDR image](https://hdrihaven.com/hdri/?c=skies&h=sunflowers) found on HDRI Haven.  Using the 4k resolution HDRI, I first tried to generate with defaults:

cli.exe -inputPath sunflowers_4k.hdr

Here is the output returned:
```
inputPath set to C:\dev\android\projects\artwork\hdri\sunflowers\sunflowers_4k.hdr
outCubeMap set to outputCubeMap.ktx2
outLUT set to outputLUT.png
sampleCount set to 1024
mipLevelCount set to 10
targetFormat set to R16G16B16A16_SFLOAT
distribution set to GGX
lodBias set to 0.000000
inputIsCubeMap flag is set to False
debug flag is set to False
Vulkan instance created
Physical Device created: Intel(R) UHD Graphics 620
APIVersion: 1.1.82
DriverVersion: 1644686
Vulkan pipeline cache loaded
Successfully loaded sunflowers_4k.hdr 4096 x 2048 x 3
Transform panorama image to cube map
Generating mipmap levels
Filtering GGX
Failed to submit queue [4294967292]
Failed to download Image
Stored pipeline.cache [77kb]
```
The cli fails in downloadCubemap, during the call to executeCommandBuffer where all copyImage2DToBuffer commands are submitted to the graphics queue.  Specifically, `vkQueueSubmit` returns VK_ERROR_DEVICE_LOST.

I also tried this on a older NVidia Quadro with similar results.

The only workaround I've been able to find is to explicitly set the cubemap resolution to 256.

**Comment by abwood** [comment]
Well, I'll close this.  After some Intel driver updates it appears this issue has resolved itself.  Perhaps my NVidia driver on my other test machine is also dated.  Having two different machines fail seemed to odd to suspect the drivers.

**Comment by abwood** [comment]
I'll leave this closed, but my earlier success was a fluke.  I've determined that the issue is that Filtering GGX operation was taking too long on my Intel 620, and would terminate after exceeding the TDR delay.  To verify, I updated my TDRDelay to 1000 seconds (intentionally large) and let it run with defaults.  It took a couple of minutes to finish (!), but it did ultimately complete successfully with the default settings.

Perhaps the Readme should make a note of this potential error and offer the workaround of adjusting the TDRDelay ... or reducing the sampleCount/mipLevel/cubeMapResolution.
